src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -64,6 +64,20 @@ namespace ts.server.typingsInstaller {
         onRequestCompleted: RequestCompletedAction;
     }
 
+    function isPackageOrBowerJson(fileName: string) {
+        const base = getBaseFileName(fileName);
+        return base === "package.json" || base === "bower.json";

```

Comment:
Should this take a flag for case sensitivity?

Reply:
the file here is always canonical file name